### PR TITLE
[Config]Add uri param to demonstrate tablet meta content of byte type for debug

### PR DESCRIPTION
**Pull request by xinghuayu007**
## Proposed changes

When get a table meta content by a http interface, Doris will transfer tablet meta into json string. If a field is byte type, it wiil be enconded into base64 code, like Pic1 Field `default:"MTA="`. 
2021-04-22 17-28-33屏幕截图

It is unreadble for debug. Therefore we add a uri param `enable_byte_to_base64` to support not transfer byte into base64 code.  Look at Pic2, Field `default : "10"` is readble. For example, `localhost:8040/api/meta/header/13310/385479563?enable_byte_base64=false`
2021-04-22 18-20-48屏幕截图

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If these changes need document changes, I have updated the document
- [x] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/http/action/meta_action.cpp [hunk 1]
```diff
@@ -63,6 +63,8 @@ Status MetaAction::_handle_header(HttpRequest* req, std::string* json_meta) {
     tablet->generate_tablet_meta_copy(tablet_meta);
     json2pb::Pb2JsonOptions json_options;
     json_options.pretty_json = true;
+    // default value of bytes_to_base64 is true, compatible with previous config
+    json_options.bytes_to_base64 = config::enable_byte_to_base64;
```
Comment (acelyc111) [inline be/src/http/action/meta_action.cpp:67]: How about make it as a uri parameter, it would be more flexible?

Comment (xinghuayu007) [reply be/src/http/action/meta_action.cpp]: good idea!

**Comment by morningman** [comment]
Why not just print the readable value? The previous base64 display seems useless.

**Comment by xinghuayu007** [comment]
> Why not just print the readable value? The previous base64 display seems useless.

It has been modified to be default false to disable base64.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**